packages/react-table/src/components/Table/examples/Table.md

```diff
@@ -365,6 +369,159 @@ class SortableTable extends React.Component {
 }
 ```
 
+### Sortable with toolbar
+
+Sorting a table may also be controlled with a toolbar. This is useful for small screen sizes, when the table will responsively shrink into a compact form that displays column headers per row and selecting a column header will not work to sort a table.
+
+```js
+import React from 'react';
+import {
+  Table,
+  TableHeader,
+  TableBody,
+  sortable,
+  SortByDirection,
+  wrappable,
+  cellWidth,
+  info
+} from '@patternfly/react-table';
+import {
+  Toolbar,
+  ToolbarItem,
+  ToolbarContent,
+  OptionsMenu,
+  OptionsMenuItemGroup,
+  OptionsMenuItem,
+  OptionsMenuSeparator,
+  OptionsMenuToggle
+} from '@patternfly/react-core';
+import SortAmountDownIcon from '@patternfly/react-icons/dist/esm/icons/sort-amount-down-icon';
+
+class SortableTable extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      columns: [
+        { title: 'Repositories', transforms: [sortable] },
+        {
+          title: 'Branches',
+          transforms: [
+            info({
+              tooltip: 'More information about branches'
+            }),
+            sortable
+          ]
+        },
+        { title: 'Pull requests', transforms: [sortable] },
+        {
+          title: 'Workspaces',
+          transforms: [sortable]
+        },
+        {
+          title: 'Last commit',
+          transforms: [
+            info({
+              tooltip: 'More information about commits'
+            })
+          ]
+        }
+      ],
+      rows: [
+        ['one', 'two', 'a', 'four', 'five'],
+        ['a', 'two', 'k', 'four', 'five'],
+        ['p', 'two', 'b', 'four', 'five']
+      ],
+      sortBy: {},
+      isSortDropdownOpen: false
+    };
+
+    this.onSort = this.onSort.bind(this);
+    this.onToggle = this.onToggle.bind(this);
+  }
+
+  onSort(_event, index, direction) {
+    const sortedRows = this.state.rows.sort((a, b) => (a[index] < b[index] ? -1 : a[index] > b[index] ? 1 : 0));
+    this.setState({
+      sortBy: {
+        index,
+        direction
+      },
+      rows: direction === SortByDirection.asc ? sortedRows : sortedRows.reverse()
+    });
+  }
+
+  onToggle(isSortDropdownOpen) {
+    this.setState({
+      isSortDropdownOpen
+    });
+  }
+
+  render() {
+    const { columns, rows, sortBy, isSortDropdownOpen } = this.state;
+
+    return (
+      <React.Fragment>
+        <Toolbar id="toolbar">
+          <ToolbarContent>
+            <ToolbarItem>
+              <OptionsMenu
```

Comment:
OptionsMenu! Whew! Haven't seen one of those in a while - would it be better to use a MenuToggle and Menu?

Reply:
Core uses options menu. WDYT @mcoker @mattnolting

Reply:
I think in this demo we should match what core has (options menu).